network/interfaces.go

```diff
@@ -92,13 +92,17 @@ type RequestHandler func(context.Context, Request) (Response, error)
 // HostNetwork simple interface to send network requests and process network responses.
 //go:generate minimock -i github.com/insolar/insolar/network.HostNetwork -o ../testutils/network -s _mock.go
 type HostNetwork interface {
+	component.Starter
+	component.Stopper
 	// PublicAddress returns public address that can be published for all nodes.
 	PublicAddress() string
 	// GetNodeID get current node ID.
 	GetNodeID() insolar.Reference
 
 	// SendRequest send request to a remote node.
 	SendRequest(ctx context.Context, request Request, receiver insolar.Reference) (Future, error)
+	// SendRequestPacket send request packet to a remote node.
+	SendRequestPacket(ctx context.Context, request Request, receiver *host.Host) (Future, error)
```

Comment:
Maybe `SendRequestToHost`? Both functions above send request packets, this name does not fit imho